### DISCUSSION TITLE
Open Telemetry

### DISCUSSION THREAD

**Discussion by bmallya** [vectordotdev/vector#18851] [General]
Hello,
Can vector support forwarding logs/events to an open telemetry collector? If so, is there an example ?

Thanks

**Comment by jszwedko** [comment]
Hi! Not via OTLP but you might be able to wire together another Vector sink with an OpenTelemetry receiver. OTLP support is being tracked by https://github.com/vectordotdev/vector/issues/13622

**Reply by bmallya** [reply]
Are you saying something like this?
source -> sink(vector using grpc) -> oltp receiver?

**Reply by jszwedko** [reply]
I don't think that would work, no, since the `vector` sink doesn't emit OTLP. It looks like you might be able to use the `splunk_hec` Vector sink with the `splunkhecreceiver` OpenTelemetry collector receiver.

**Reply by bmallya** [reply]
Thanks @jszwedko , let me give this a try.
